src/file-input.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @flow
+ */
+// On Android we need a polyfill for Symbol
+import Symbol from 'es6-symbol/polyfill'
```

Comment:
I havn't used symbols before, but is it really necessary to use them here? This polyfill looks to be as big as `apollo-upload-client` itself.